microsoft-365/admin/misc/icann-verification-of-contact-information.md

```diff
@@ -22,7 +22,7 @@ ROBOTS: NOINDEX
 description: "Learn how to validate your contact information to comply with ICANN regulations."
 ---
 
-# ICANN Verification of Contact Information
+# ICANN verification of contact information
```

Comment:
@lhcole613 - Same comment as above about adding product name. Should this be "ICANN verification of contact information in Microsoft 365 for business" or whichever product this is in (I'm not sure from this context)?